libs/sdk-ui-dashboard/src/model/commandHandlers/alerts/updateAlertHandler.ts

```diff
@@ -21,28 +20,13 @@ export function* updateAlertHandler(ctx: DashboardContext, cmd: UpdateAlert): Sa
     // eslint-disable-next-line no-console
     console.debug("handling update alert", cmd, "in context", ctx);
 
-    try {
-        const alert: PromiseFnReturnType<typeof updateAlert> = yield call(
-            updateAlert,
-            ctx,
-            cmd.payload.alert,
-        );
+    const alert: PromiseFnReturnType<typeof updateAlert> = yield call(updateAlert, ctx, cmd.payload.alert);
```

Comment:
Why the removal of the try catch?

Reply:
Base of discussion with lupko that is not necessary add global try/catch into handler its moved into generic handler. 
Going to create separate commit for that.